libtenzir/src/index.cpp

```diff
@@ -1371,6 +1372,10 @@ index(index_actor::stateful_pointer<index_state> self,
     // Flush buffered batches and end stream.
     self->state.stage->stop();
   });
+  detail::weak_run_delayed_loop(self, std::chrono::seconds{10}, [self] {
+    TENZIR_VERBOSE("{} has {} active stream managers", *self,
+                   self->stream_managers().size());
+  });
```

Comment:
```suggestion
```

By itself, this message provides too little information to aid understanding. I'd say it's rather adding more confusion for everybody that was not present during our conversation.
What would help more is printing all `stream_managers` that do not belong to the list of current active partitions if there are any. But since we know that we are the only ones creating those outbound stream slots I wouldn't bother.